### PR TITLE
Change PolicySet API version to v1beta1

### PR DESCRIPTION
**Pull request by ycao56**
https://github.com/stolostron/backlog/issues/20475
Signed-off-by: Yu Cao <ycao@redhat.com>

### CODE REVIEW

File: internal/plugin.go [hunk 1]
```diff
@@ -18,7 +18,9 @@ import (
 
 const (
 	configPolicyKind           = "ConfigurationPolicy"
-	policyAPIVersion           = "policy.open-cluster-management.io/v1"
+	policyAPIGroup             = "policy.open-cluster-management.io"
+	v1                         = "v1"
+	v1beta1                    = "v1beta1"
```
Comment (dhaiducek) [inline internal/plugin.go:23]: I'm not sure I see the benefit of naming the variable names the same as their content. Would it make more sense to call them `policyAPIVersion` and `policySetAPIVersion`? And then we can change the content without needing to update the variable name.

Comment (ycao56) [reply internal/plugin.go:23]: This would save us two split calls on line 1147 and 1156

Comment (ycao56) [reply internal/plugin.go:23]: Let me update. I can make it variable too

**Review by dhaiducek — APPROVED** [review]
That works! Sorry, I should have been clearer--I was thinking more like `policyAPIVersion=v1`, but this works too (and is probably even cleaner! 🙂

**Pull request merged**